src/index.ts

```diff
@@ -35,6 +35,11 @@ export interface TCPOptions {
    * https://nodejs.org/api/net.html#servermaxconnections
    */
   maxConnections?: number
+
+  /**
+   * Close server (stop listening for new connection) if connections exceed a limit.
```

Comment:
Might be worth also mentioning the other side too, otherwise lay reader may see this option as a one-way valve.

```suggestion
   * Close server (stop listening for new connections) if connections exceed a limit.
   * Open server (start listening for new connections) if connections fall below a limit.
```